### ISSUE TITLE
Create option to set hyphen char in LuaTeX

### ISSUE THREAD

**Issue by typiconman** [slavonic/cu-tex#72]
From my discussion with @pgmmpk:
Hyphenation char in LuaTeX can be set using:
\textchurchslavonic{\prehyphenchar=`\-}%
in the body of the document.

By default the hyphenation character should be the hyphen, and if an underscore is desired, it should be set using an option, which invokes the above code in LuaTeX.
In XeTeX this is set by specifying HyphenChar when loading the font.

Alternatively, we should describe all of this in the documentation if we don't create the options.

**Comment by YNKTN** [comment]
Просмотром в FontForge обнаружил что в вашем пакете шрифтов символ переноса HyphenChar ( - ) имеет стилевую альтернативу ( _ )  в субтаблице ss01 "churchslavonic hyphenation".
Таким образом, чтобы переключить HyphenChar ( - ) на ( _ ) нужно в пакете fontspec задать следующие параметры: RawFeature=+ss01

(After reviewing it in FontForge, I discovered that in your font package, HyphenChar ( - ) has a style alternative ( _ ) in the ss01 "churchslavonic hyphenation" subtable.
Therefore, to switch HyphenChar ( - ) to ( _ ), you need to set the following parameters in the fontspec package: RawFeature=+ss01)

пример: (example:)
\usepackage{fontspec}
\setmainfont[RawFeature=+ss01]{PochaevskUnicode.otf}

Способ универсален для lualatex и xelatex (The method is universal for lualatex and xelatex)

**Comment by typiconman** [comment]
Да, это во всех наших шрифтах, как workaround для программ, в которых нельзя указать символ переноса (напр., Ворд, LibreOffice). Можно и в TeX так делать.
@pgmmpk давайте внесем в документацию.
